ign_rviz_plugins/res/qml/QoSConfig.qml

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.3
+import QtQuick.Controls.Material 2.1
+
+ColumnLayout {
+  id: qosConfig
+  signal profileUpdate(int depth, int history, int reliability, int durability);
+
+  width: parent.width
+  Layout.fillWidth: true
+
+  RoundButton {
+    Layout.fillWidth: true
+    Layout.preferredHeight: 30
+    text: configColumn.visible ? "\u25B4" : "\u25BE"
+    Material.background: "#fafafa"
+    onClicked: {
+      configColumn.visible = !configColumn.visible;
+    }
+  }
+  
+  ColumnLayout {
+    id: configColumn
+    Layout.fillWidth: true
+    visible: false
+
+    RowLayout {
+      width: parent.width
+
+      Text {
+        width: 75
+        Layout.minimumWidth: 75
+        anchors.left: parent.left
+        anchors.leftMargin: 2
+        text: "Depth"
+        font.pointSize: 10.5
+      }
+
+      TextField {
+        id: depthText
+        Layout.fillWidth: true
+        text: "5"
+        validator: RegExpValidator {
+          regExp: /[0-9]+/ig
+        }
+        onEditingFinished: {
+          qosConfig.profileUpdate(5, historyCombo.currentIndex, reliabilityCombo.currentIndex, durabilityCombo.currentIndex)
```

Comment:
depth is always 5 in all the calls. Why?

Reply:
My bad. I'll fix this.